scripts/build/build.ps1

```diff
@@ -34,7 +34,18 @@ header "Compiling"
 _ "$RepoRoot\scripts\compile\compile.ps1" @("$Configuration")
 
 header "Running Tests"
-_ "$RepoRoot\scripts\test\runtests.ps1"
+# Capture PATH for later
```

Comment:
If you want to preserve the path, this is not the right place for it. We should be saving & restoring the path in build.cmd, not just specifically around tests. This pattern will riddle the build scripts with similar save & restore stuff.